wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/TemporaryWorkspaceRemover.java

```diff
@@ -43,7 +43,9 @@ public TemporaryWorkspaceRemover(WorkspaceDao workspaceDao, WorkspaceRuntimes ru
     this.runtimes = runtimes;
   }
 
-  @PostConstruct
+  @ScheduleDelay(
+      initialDelayParameterName = "che.workspace.cleanup_temporary_stopped_period_min",
```

Comment:
isn't it 60 seconds now? I think you should set `unit`

Reply:
d'ouh. correct.